src/utils/plugins.js

```diff
@@ -1,6 +1,31 @@
+import OurVue from 'vue'
+import warn from './warn'
 import { setConfig } from './config'
 import { hasWindowSupport } from './env'
 
+const MULTIPLE_VUE_WARNING = `Multiple instances of Vue detected! See:
```

Comment:
This can be a short link to wiki or docs  :)

Reply:
So maybe just a single link to the [module bund'ers](https://bootstrap-vue.js.org/docs#using-module-bundlers) section of the docs (which mentions the use of aliases)

Reply:
Changed warning message to just one link to the docs getting started page.